manifests/overlays/rhoai/params.env

```diff
@@ -0,0 +1,2 @@
+odh-dashboard-image=quay.io/opendatahub/odh-dashboard:main
+# This is needed to inject the dashboard image from the CSV to the Deployment.
```

Comment:
```suggestion
# This will be overridden through the Operator as needed to inject the dashboard image as a variable to be used in the manifests (look for references of this variable)
odh-dashboard-image=quay.io/opendatahub/odh-dashboard:main
```
Best to avoid specifics of use-cases since it's just a variable.

Reply:
Also, should we bother with making this `rhoai` specific and just go with general variables the operator can override if they choose?

Reply:
@zdtsw What variables does the operator use for both ODH & RHOAI, can we just make a mapping at the root or something for all of them and just override as needed?

Reply:
we can use any "key" you set in the params.env file
currently we are using `odh-dashboard-image` but if you want to change it to "dashboard-image" we can  update in operator too.

Reply:
And I've already updated the pr with the changes 👌